docs/source/hooks/introduction.md

```diff
@@ -82,7 +82,7 @@ class DataCatalogHooks:
         return logging.getLogger(self.__class__.__name__)
 
     @hook_impl
-    def after_catalog_created(self, catalog: DataCatalog) -> None:
+    def after_catalog_created(self, catalog: DataCatalog, **kwargs) -> None:
```

Comment:
What is this for?

Reply:
It was my understanding that the Hook spec contains extra parameters:

https://github.com/kedro-org/kedro/blob/3439e72ef9ef79095da35db88d0584e684299e09/kedro/framework/hooks/specs.py#L20-L29

but if these are not needed, I'd love to know why 😅

Reply:
My understanding is you only need to define it if you want to access it, it's not necessary to define all arguments.

Reply:
It's written in the docs https://pluggy.readthedocs.io/en/stable/#opt-in-arguments

> To allow for hookspecs to evolve over the lifetime of a project, hookimpls can accept less arguments then defined in the spec. This allows for extending hook arguments (and thus semantics) without breaking existing hookimpls.

I will revert this change.